FakeTrello/Models/Card.cs

```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Web;
+
+namespace FakeTrello.Models
+{
+    public class Card
+    {
+        [Key]
+        public int CardId { get; set; }
+
+        public string Title { get; set; }
+
+        public string Description { get; set; }
```

Comment:
Does your card need a link to its containing list? if so you should add the navigation property. It's not necessary for Entity Framework when it is building your db, it's more of a convenience thing.